src/components/Heatmap/heatmap.less

```diff
@@ -4,12 +4,24 @@ text {
     font-family: "Source Sans Pro", sans-serif !important;
 }
 
-.heatmap svg {
+.heatmap {
+    overflow-x: auto;
+    width: ~"calc(100vw - 64px)";
+    margin: 0 auto;
     position: relative;
     left: 50%;
-    -webkit-transform: translateX(-50%);
-    -ms-transform: translateX(-50%);
     transform: translateX(-50%);
+    position: relative;
+    text-align: center;
+}
+
+.legend {
+    width: 300px;
```

Comment:
Container needs to be a little bit bigger than the legend itself, at least 16px, for the SVG overflow.